src/api/db/index.js

```diff
@@ -92,6 +99,17 @@ export default function initBookshelf(config) {
         }
       }
 
+      let schools = this.schools();
+      schoolNames = await School.fetchAll();
+      schoolNames = schoolNames.serialize().map(row => row.name);
+
+      for (let name of labelNamesToAdd) {
+        if (schoolNames.indexOf(name) > -1) {
+          schoolNamesToKeep.push(name);
+          _.pull(labelNamesToAdd, name);
+        }
+      }
+
```

Comment:
Now if a tag is a school, it gets added to `schoolNamesToKeep` and removed from `labelNamesToAdd` before label and school relations are saved.